### PR TITLE
Empty deploy

### PR DESCRIPTION
**Pull request by seanyoung**

### CODE REVIEW

File: deploy/jobs/jobs_contracts.go [hunk 1]
```diff
@@ -125,6 +125,9 @@ func DeployJob(deploy *def.Deploy, do *def.DeployArgs, client *def.Client, resp
 		if binaryResponse.Error != "" {
 			return "", fmt.Errorf("Something went wrong when you were trying to link your binaries: %v", binaryResponse.Error)
 		}
+		if binaryResponse.Binary == "" {
+			return "", fmt.Errorf("Contract binary code missing")
```
Comment (silasdavis) [inline deploy/jobs/jobs_contracts.go]: Can we add something like "Is your contract an abstract interface? You may not have unintentionally not implemented all abstracted inherited methods."

File: deploy/jobs/jobs_contracts.go [hunk 2]
```diff
@@ -187,10 +193,14 @@ func DeployJob(deploy *def.Deploy, do *def.DeployArgs, client *def.Client, resp
 				if err != nil {
 					return "", err
 				}
+				deployedCount++
 				if strings.ToLower(response.Objectname) == strings.ToLower(strings.TrimSuffix(filepath.Base(deploy.Contract), filepath.Ext(filepath.Base(deploy.Contract)))) {
 					baseObj = result
 				}
 			}
+			if deployedCount == 0 {
+				return "", fmt.Errorf("Contract binary code missing")
```
Comment (silasdavis) [inline deploy/jobs/jobs_contracts.go]: maybe share error

**Comment by silasdavis** [comment]
@seanyoung looks good other than making error messages a but more helpful

File: deploy/jobs/jobs_contracts.go (hunk 1 above)
Comment (seanyoung) [reply deploy/jobs/jobs_contracts.go]: Good point, done

File: deploy/jobs/jobs_contracts.go (hunk 2 above)
Comment (seanyoung) [reply deploy/jobs/jobs_contracts.go]: also true. done

**Pull request merged**